### PR TITLE
Code updates related to Jackson v1.3.0

### PR DESCRIPTION
**Pull request by niwsa**
Tested the changes with new release of `@boxyhq/remix-auth-sso` npm package.

### CODE REVIEW

**Comment by devkiran** [comment]
@niwsa I'm getting the following error while running.

Steps

- npm install
- npm run dev
- Visit the URL http://localhost:3366/

```
TypeError: Cannot read properties of undefined (reading 'split')
    at Object.<anonymous> (/Users/kiran/Desktop/boxyhq/jackson-remix-auth/app/sessions.server.ts:10:42)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at /Users/kiran/Desktop/boxyhq/jackson-remix-auth/node_modules/@remix-run/serve/dist/index.js:43:17
    at Layer.handle [as handle_request] (/Users/kiran/Desktop/boxyhq/jackson-remix-auth/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/kiran/Desktop/boxyhq/jackson-remix-auth/node_modules/express/lib/router/route.js:144:13)
```

**Comment by niwsa** [comment]
You'll have to setup the env from example file. I will update the README.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,53 @@
...
+```
+npm i
+```
+
+## Set up the env
+
+Create a `.env` file from `.env.example`
+
+```
+cp .example.env .env
```
Comment (devkiran) [inline README.md]: should be `.env.example`

File: app/auth.jackson.server.ts [hunk 2]
```diff
@@ -1,31 +1,41 @@
...
 // 📄 https://boxyhq.com/docs/jackson/deploy/env-variables
 const opts: JacksonOption = {
-  externalUrl: "", // APP BASE URL
+  externalUrl: "",
   samlPath: "/api/oauth/saml",
+  oidcPath: "/api/oauth/oidc",
   samlAudience: "",
   db: {
     engine: "sql",
     url: "postgresql://postgres:postgres@localhost:5432/postgres",
```
Comment (devkiran) [inline app/auth.jackson.server.ts]: can you read the database url from the `.env`?

**Review by devkiran — APPROVED** [review]

**Pull request merged**